### PR TITLE
Attach msi to release on github.com

### PR DESCRIPTION
**Pull request by thedaniel**
I thiiiink this is all that's necessary now that the msi is generated.

### CODE REVIEW

**Comment by thedaniel** [comment]
I think maybe specs on master are flaky right now, so I'll re-run the build on the failing CI host a few times and see if it greens.

**Comment by bittin** [comment]
Cool :)

File: spec/fixtures/sample.txt [hunk 1]
```diff
@@ -1 +1 @@
-Some textSome textSome text.
+Some textSome textSome textSome text.
```
Comment (50Wliu) [inline spec/fixtures/sample.txt]: :scream:

Comment (thedaniel) [inline spec/fixtures/sample.txt]: what the heck??

**Pull request merged**